docs/advanced_topics/privacy.md

```diff
@@ -8,6 +8,20 @@ Users with publish permission on a page can set it to be private by clicking the
 -   **Accessible with a shared password:** The user must enter the given shared password to view the page. This is appropriate for situations where you want to share a page with a trusted group of people, but giving them individual user accounts would be overkill. The same password is shared between all users, and this works independently of any user accounts that exist on the site.
 -   **Accessible to users in specific groups:** The user must be logged in, and a member of one or more of the specified groups, in order to view the page.
 
+```{warning}
+Shared passwords should not be used to protect sensitive content, as the password is shared between all users, and stored in plain text in the database. Where possible, it's recommended to require users log in to access private page content.
+```
+
+You can disable shared password for pages using `WAGTAIL_ALLOW_SHARED_PASSWORD_PAGE`:
```

Comment:
```suggestion
You can disable shared password for pages using `WAGTAIL_ALLOW_SHARED_PASSWORD_PAGE`.

Any existing shared password usage will remain active but will not be viewable by the user within the admin.

```

Reply:
This added detail is quite important - should we elevate it to a warning popout?

Reply:
I'm concerned about admonition overload, we really should not use warnings too much. 

The content directly above this is a Warning.

Reply:
How easy is it to report (ideally through the admin) on which pages are password restricted? I'm concerned that people might not realise some pages are still protected like this.